README.md

```diff
@@ -61,10 +61,39 @@ This project is in active development and not ready for production use. However,
 pip install uarray
 ```
 
+or
+
 ```bash
 conda install -c conda-forge -c uarray uarray
 ```
 
+
+## Development
+
+Make a fork of the main [uarray repository](https://github.com/Quansight-Labs/uarray) and clone the fork:
+
+```
+git clone https://github.com/<your-github-username>/uarray
+```
+
+### Install
+
+`uarray` and all development dependencies can be installed via:
+
+```
+pip install -e ".[all]"
+```
+
+Note that uarray supports Python versions >= 3.5.
```

Comment:
We need to tell people here that it currently requires `contextvars` to be installed manually for Python < 3.7.

Reply:
I pushed to `master` so that this isn't required anymore.